docs/Salesforce-batchsink.md

```diff
@@ -17,7 +17,7 @@ Configuration
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
+**Security Token:** Salesforce security Token. If the password does not contain the security token, the plugin 
 will append the token before authenticating with salesforce.
```

Comment:
Can you change salesforce to Salesforce?

Reply:
Done